ansible/roles/test/files/ptftests/router_adv_mflag_test.py

```diff
@@ -0,0 +1,242 @@
+        masked_rapkt.set_do_not_care_scapy(RA, "prf")
+        masked_rapkt.set_do_not_care_scapy(RA, "P")
+        masked_rapkt.set_do_not_care_scapy(RA, "res")
+        masked_rapkt.set_do_not_care_scapy(RA, "routerlifetime")
+        masked_rapkt.set_do_not_care_scapy(RA, "reachabletime")
+        masked_rapkt.set_do_not_care_scapy(RA, "retranstimer")
+
+        masked_rapkt.set_do_not_care_scapy(PrefixInfo, "prefixlen")
+        masked_rapkt.set_do_not_care_scapy(PrefixInfo, "L")
+        masked_rapkt.set_do_not_care_scapy(PrefixInfo, "A")
+        masked_rapkt.set_do_not_care_scapy(PrefixInfo, "R")
+        masked_rapkt.set_do_not_care_scapy(PrefixInfo, "res1")
+        masked_rapkt.set_do_not_care_scapy(PrefixInfo, "validlifetime")
+        masked_rapkt.set_do_not_care_scapy(PrefixInfo, "preferredlifetime")
+        masked_rapkt.set_do_not_care_scapy(PrefixInfo, "res2")
+        masked_rapkt.set_do_not_care_scapy(PrefixInfo, "prefix")
+
+        return masked_rapkt
+
+"""
+@summary: This test is to validate the unsolicted router advertisements sent on
+the VLAN network of the ToR. In this test we listen on the first PTF port
+(Eg eth0) for any RA messages.
+
+"""
+
+class RadvUnSolicitedRATest(DataplaneBaseTest):
+    def __init__(self):
+        DataplaneBaseTest.__init__(self)
+
+    def setUp(self):
+        DataplaneBaseTest.setUp(self)
+
+        self.test_params = testutils.test_params_get()
+
+        self.hostname = self.test_params['hostname']
+        self.downlink_vlan_mac = self.test_params['downlink_vlan_mac']
+        self.downlink_vlan_ip6 = self.test_params['downlink_vlan_ip6']
+        self.ptf_port_index = int(self.test_params['ptf_port_index'])
+        self.ptf_port_mac = self.dataplane.get_mac(0, self.ptf_port_index)
+        self.radv_max_ra_interval = int(self.test_params['max_ra_interval'])
+
+        rapkt = self.create_icmpv6_router_advertisement_packet_send(
+                                                src_mac=self.downlink_vlan_mac,
+                                                dst_mac=ALL_NODES_MULTICAST_MAC_ADDRESS,
+                                                src_ip=self.downlink_vlan_ip6,
+                                                dst_ip=ALL_NODES_IPV6_MULTICAST_ADDRESS)
+        self.masked_rapkt = self.mask_off_dont_care_ra_packet_fields(rapkt)
+
+
+    def tearDown(self):
+        DataplaneBaseTest.tearDown(self)
+
+    """
+    @summary: Verify received RA packet
+
+    """
+
+    def verify_periodic_router_advertisement_with_m_flag(self):
+        testutils.verify_packet(self,
+                                self.masked_rapkt,
+                                self.ptf_port_index,
+                                self.radv_max_ra_interval)
+        logging.info("Received unsolicited RA from:%s on PTF eth%d having M=1",
+                     self.downlink_vlan_ip6,
+                     self.ptf_port_index)
+        sk=scapy2.sniff(iface='eth1',timeout=10)
```

Comment:
Testutils outputs the packet if verify_packet fails. The logging here is not needed

Reply:
I removed those logs